plugins/woocommerce-docs/src/Manifest/ManifestProcessor.php

```diff
@@ -31,6 +31,35 @@ private static function get_converter() {
 		return $converter;
 	}
 
+	/**
+	 * Generate post args
+	 *
+	 * @param mixed $post The post to generate args for.
+	 * @param mixed $post_content The post content.
+	 * @return array
+	 */
+	private static function generate_post_args( $post, $post_content ) {
+		$possible_attributes = array(
+			'post_title',
+			'post_author',
+			'post_date',
+			'comment_status',
+		);
+
+		$args = array();
+
+		foreach ( $possible_attributes as $attribute ) {
+			if ( isset( $post[ $attribute ] ) ) {
+				$args[ $attribute ] = $post[ $attribute ];
+			}
+		}
+
+		$args['post_content'] = $post_content;
+		$args['post_status']  = 'publish';
```

Comment:
Could it make sense to move this up above the loop (and add it to `$possible_attributes`) to make this something that could also be in a `$post[ $attribute ]`, just in case we want to be able to set it to, say, `draft` for some content in the future?

Reply:
Yeah true lets do that! Good idea!

Reply:
Allowed setting the status from frontmatter here: https://github.com/woocommerce/woocommerce/pull/39214/commits/85ac3e04db0e368443f24bb5d6a2ebe4493ae7cc

Reply:
Resolved here https://github.com/woocommerce/woocommerce/pull/39214/commits/85ac3e04db0e368443f24bb5d6a2ebe4493ae7cc